spec.html

```diff
@@ -1095,15 +1095,37 @@ <h1>Identity</h1>
   </emu-clause>
 </emu-clause>
 
-<emu-clause id="sec-ecmascript-data-types-and-values" aoid="Type">
+<emu-clause id="sec-ecmascript-data-types-and-values">
   <h1>ECMAScript Data Types and Values</h1>
   <p>Algorithms within this specification manipulate values each of which has an associated type. The possible value types are exactly those defined in this clause. Types are further subclassified into ECMAScript language types and specification types.</p>
-  <p>Within this specification, the notation “Type(_x_)” is used as shorthand for “the <em>type</em> of _x_” where “type” refers to the ECMAScript language and specification types defined in this clause.</p>
 
   <emu-clause id="sec-ecmascript-language-types">
     <h1>ECMAScript Language Types</h1>
     <p>An <dfn variants="ECMAScript language types">ECMAScript language type</dfn> corresponds to values that are directly manipulated by an ECMAScript programmer using the ECMAScript language. The ECMAScript language types are Undefined, Null, Boolean, String, Symbol, Number, BigInt, and Object. An <dfn variants="ECMAScript language values">ECMAScript language value</dfn> is a value that is characterized by an ECMAScript language type.</p>
 
+    <emu-clause id="sec-type-ao" type="abstract operation">
+      <h1>
+        Type (
+          _v_: an ECMAScript language value,
+        ): one of ~undefined~, ~null~, ~boolean~, ~string~, ~symbol~, ~number~, ~bigint~, or ~object~
```

Comment:
```suggestion
        ): ~undefined~, ~null~, ~boolean~, ~string~, ~symbol~, ~number~, ~bigint~, or ~object~
```
(We don't use "one of" in this kind of context.)